docs/glossary.rst

```diff
@@ -52,6 +52,18 @@
 
        http://gunicorn.org/
 
+   IDNA
+
+       An Internationalized Domain Name in Applications (IDNA) is an
+       standard for encoding Internet Domain Names that contain in
```

Comment:
"is **a** standard"

Reply:
or maybe "is an industry standard"

Reply:
Fixed.

@samuelcolvin I'm not a native English speaker.
If you feel that documentation could be improved and have time for the job -- please don't hesitate.

Reply:
> and have time

:smile: 

I think overall it's pretty good. If I see anything jarring I'll try to correct.